spec/controllers/plugin/plugin_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe PluginController do
+  render_views
+
+  describe '#user' do
+    before do
+      create :group, :donors
+      create :group, :champions
+    end
+
+    let!(:user) { create :user_with_team }
+
+    it 'returns user data' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.